### PR TITLE
Fixes #37 | Configure consul timeouts

### LINKED ISSUE #37: Configure HTTP client timeouts
We should not wait long for an i/o timeout during sync when a mesos slave is down, etc.

### PR DESCRIPTION
**Pull request by janisz**

### CODE REVIEW

File: consul/agents.go [hunk 1]
```diff
@@ -85,6 +85,11 @@ func (a *ConcurrentAgents) createAgent(host string) (*consulapi.Client, error) {
 		config.Scheme = "https"
 	}
 
+	if a.config.Timeout != 0 {
+		log.Debugf("Seting timeout to %s", a.config.Timeout.String())
```
Comment (dankraw) [inline consul/agents.go]: typo Seting -> Setting

Comment (janisz) [inline consul/agents.go]: Fixed

File: README.md [hunk 2]
```diff
@@ -70,6 +70,7 @@ consul-ssl-cert        |                       | Path to an SSL client certifica
 consul-ssl-verify      | `true`                | Verify certificates when connecting via SSL
 consul-token           |                       | The Consul ACL token
 consul-tag             | `marathon`            | Common tag name added to every service registered in Consul, should be unique for every Marathon-cluster connected to Consul
+consul-timeout         | `1 s`                 | Max request duration
```
Comment (dankraw) [inline README.md]: `1 s` => `1s` (no space)

File: config/config.go [hunk 3]
```diff
@@ -68,6 +68,7 @@ func (config *Config) parseFlags() {
 	flag.StringVar(&config.Consul.SslCaCert, "consul-ssl-ca-cert", "", "Path to a CA certificate file, containing one or more CA certificates to use to validate the certificate sent by the Consul server to us")
 	flag.StringVar(&config.Consul.Token, "consul-token", "", "The Consul ACL token")
 	flag.StringVar(&config.Consul.Tag, "consul-tag", "marathon", "Common tag name added to every service registered in Consul, should be unique for every Marathon-cluster connected to Consul")
+	flag.DurationVar(&config.Consul.Timeout, "consul-timeout", time.Second, "Max request duration")
```
Comment (dankraw) [inline config/config.go]: I would suggest a little change: "Time limit for requests made by the Consul HTTP client. A timeout of zero means no timeout."

File: README.md (hunk 2 above)
Comment (janisz) [inline README.md]: Fixed

File: config/config.go (hunk 3 above)
Comment (janisz) [inline config/config.go]: Fixed

**Comment by dankraw** [comment]
:+1:

**Pull request merged**